### PR TITLE
Book detail 4

### PR DESCRIPTION
**Pull request by Gianni-Nicolas**
**Summary**
Add book badge in book detail

** Screenshot**

Captura de Pantalla 2020-02-10 a la(s) 16 02 13

**Trello Card**

https://github.com/wolox-training/ng-angular/compare/book-detail...book-detail-4?expand=1

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -37,9 +36,21 @@
   width: 900px;
 }
 
-.book-cover {
+.book-images {
+  background: url("../images/book-cover.png");
+  background-size: 100% 100%;
```
Comment (idcuadrowolox) [inline scss/detail.scss]: I think It's better to set width or height in images, not both

File: scss/detail.scss [hunk 2]
```diff
@@ -37,9 +36,21 @@
...
   @extend %container-book-elements-height;
   margin-right: 30px;
+  width: 245px;
+}
+
+.book-badge {
```
Comment (idcuadrowolox) [inline scss/detail.scss]: I think it's better put this image as :before selector in image container

**Review by idcuadrowolox — CHANGES_REQUESTED** [review]

**Review by idcuadrowolox — APPROVED** [review]

File: bookDetail.html [hunk 3]
```diff
@@ -1,24 +1,35 @@
...
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="css/application.css" />
+  </head>
+  <body>
+    <div class="general-container">
+      <div class="container-book">
+        <div class="book-images">
```
Comment (martinmona) [inline bookDetail.html]: I think you can do it without this tag

File: scss/detail.scss [hunk 4]
```diff
@@ -37,9 +36,26 @@
   width: 900px;
```
Comment (martinmona) [inline scss/detail.scss]: Please, add `max-width: 100%;`

File: scss/detail.scss [hunk 5]
```diff
@@ -8,7 +8,6 @@
 %container-book-elements-height {
   height: 368px;
   max-height: 100%;
```
Comment (martinmona) [inline scss/detail.scss]: Delete this

**Review by martinmona — CHANGES_REQUESTED** [review]

File: scss/detail.scss [hunk 6]
```diff
@@ -33,18 +29,33 @@
   @extend %display-config-vertical;
   height: 416px;
   justify-content: flex-start;
+  max-width: 100%;
   padding: 40px;
+  position: relative;
   width: 900px;
+
+  &::after {
+    background: url(../images/badge.png) no-repeat;
```
Comment (martinmona) [inline scss/detail.scss]: The correct way is using `background-image` property

File: scss/detail.scss [hunk 7]
```diff
@@ -33,18 +29,33 @@
   @extend %display-config-vertical;
   height: 416px;
   justify-content: flex-start;
```
Comment (martinmona) [inline scss/detail.scss]: That's the value by default, so delete this.

File: scss/detail.scss [hunk 8]
```diff
@@ -33,18 +29,33 @@
...
+  height: $container-book-elements-height;
   margin-right: 30px;
+  width: 245px;
 }
 
 .book-information {
   display: flex;
-  @extend %container-book-elements-height;
+  height: $container-book-elements-height;
   flex-flow: column nowrap;
```
Comment (martinmona) [inline scss/detail.scss]:
```suggestion
  flex-direction: column;
```

**Review by martinmona — CHANGES_REQUESTED** [review]

**Review by martinmona — APPROVED** [review]

**Review by nvescio — APPROVED** [review]

**Pull request merged**